cmd/thanos/compact.go

```diff
@@ -190,11 +195,17 @@ func runCompact(
 	}, func() float64 {
 		return consistencyDelay.Seconds()
 	})
+	deleteDelayMetric := prometheus.NewGaugeFunc(prometheus.GaugeOpts{
+		Name: "thanos_delete_delay_seconds",
+		Help: "Configured delete delay in seconds.",
+	}, func() float64 {
+		return deleteDelay.Seconds()
```

Comment:
Thanks :+1:

Reply:
@bwplotka Do we need to expose this configuration value as a metric?